### PR TITLE
CLDSRV-485: website redirect from custom error document

### PR DESCRIPTION
**Pull request by BourgoisMickael**
If there is a redirect on the error page object (metadata `x-amz-website-redirect-location`), when that page is returned (because of an error, and not a direct access to `error.html`) the response for a GET should be the redirect `301` but with the body of the object.

Right now the redirect is ignored on error page.

This PR needs https://github.com/scality/Arsenal/pull/2207

### CODE REVIEW

File: tests/functional/aws-node-sdk/lib/utility/website-util.js [hunk 1]
```diff
@@ -216,6 +217,25 @@ function _assertResponseHtmlRedirect(response, type, redirectUrl, method) {
         'Best redirect link ever');
         _assertResponseHtml(response.body, 'h1',
         'Welcome to your redirection file');
+    } else if (type.startsWith('redirect-error')) {
+        assert.strictEqual(response.statusCode,
+            type === 'redirect-error-found' ? 302 : 301);
```
Comment (BourgoisMickael) [inline tests/functional/aws-node-sdk/lib/utility/website-util.js:222]: `redirect-error-found` is in preparation for the other redirect error type for CLDSRV-489 it will be used in another PR #5520

File: tests/functional/aws-node-sdk/lib/utility/website-util.js [hunk 2]
```diff
@@ -216,6 +217,25 @@ function _assertResponseHtmlRedirect(response, type, redirectUrl, method) {
...
+        }
+
+        if (type === 'redirect-error-found') {
+            _assertContainsHtml(response.body);
+            _assertResponseHtml(response.body, 'title', '302 Moved Temporarily');
+            _assertResponseHtml(response.body, 'h1', '302 Moved Temporarily');
+            _assertResponseHtml(response.body, 'ul', [
+                'Code: Found',
+                'Message: Resource Found',
+            ]);
```
Comment (BourgoisMickael) [inline tests/functional/aws-node-sdk/lib/utility/website-util.js:235]: `redirect-error-found` is in preparation for the other redirect error type for CLDSRV-489 it will be used in another PR

**Review by jonathan-gramain — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
   "homepage": "https://github.com/scality/S3#readme",
   "dependencies": {
     "@hapi/joi": "^17.1.0",
-    "arsenal": "git+https://github.com/scality/arsenal#7.10.53",
+    "arsenal": "git+https://github.com/scality/arsenal#1ced481",
```
Comment (ghost) [inline package.json]: Just a remainder to use a proper tag before merging

**Review by ghost — APPROVED** [review]
